packages/eas-cli/src/ora.ts

```diff
@@ -0,0 +1,74 @@
+import oraReal from 'ora';
+
+import Log from './log';
+
+/**
+ * A custom ora spinner that sends the stream to stdout in CI, or non-TTY, instead of stderr (the default).
+ *
+ * @param options
+ * @returns
+ */
+export function ora(options?: oraReal.Options | string): oraReal.Ora {
+  const inputOptions = typeof options === 'string' ? { text: options } : options || {};
+  const disabled = Log.isDebug || !process.stdin.isTTY;
+  const ora = oraReal({
+    // Ensure our non-interactive mode emulates CI mode.
+    isEnabled: !disabled,
+    // In non-interactive mode, send the stream to stdout so it prevents looking like an error.
+    stream: disabled ? process.stdout : process.stderr,
+    ...inputOptions,
+  });
+
+  // eslint-disable-next-line no-console
+  const logReal = console.log;
+  // eslint-disable-next-line no-console
+  const infoReal = console.info;
+  // eslint-disable-next-line no-console
+  const warnReal = console.warn;
+  // eslint-disable-next-line no-console
+  const errorReal = console.error;
+
+  const oraStop = ora.stop.bind(ora);
+
+  const origStopAndPersist = ora.stopAndPersist.bind(ora);
+
+  const logWrap = (method: any, args: any[]) => {
+    oraStop();
+    method(...args);
+    ora!.start();
```

Comment:
```suggestion
    ora.start();
```
I checked out your branch locally and verified that `!` is not necessary.